src/node.cc

```diff
@@ -251,7 +251,7 @@ static void PrintErrorString(const char* format, ...) {
   vsprintf(out.data(), format, ap);
 
   // Get required wide buffer size
-  n = MultiByteToWideChar(CP_UTF8, 0, out.data(), -1, nullptr, 0);
+  n = MultiByteToWideChar(CP_UTF8, 0, out.data(), n, nullptr, 0);
```

Comment:
OK, agreed, I'll make it simpler. (I wanted to avoid allocating for the unused wide null character, but it's not worth it if it makes the code more complicated)